staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -263,27 +364,24 @@ type draPlugin struct {
 }
 
 // Start sets up two gRPC servers (one for registration, one for the DRA node
-// client). By default, all APIs implemented by the nodeServer get registered.
+// client) and implements them by calling a [DRAPlugin] implementation.
 //
 // The context and/or DRAPlugin.Stop can be used to stop all background activity.
 // Stop also blocks. A logger can be stored in the context to add values or
 // a name to all log entries.
 //
 // If the plugin will be used to publish resources, [KubeClient] and [NodeName]
-// options are mandatory.
-//
-// The DRA driver decides which gRPC interfaces it implements. At least one
-// implementation of [drapbv1alpha4.NodeServer] or [drapbv1beta1.DRAPluginServer]
-// is required. Implementing drapbv1beta1.DRAPluginServer is recommended for
-// DRA driver targeting Kubernetes >= 1.32. To be compatible with Kubernetes 1.31,
-// DRA drivers must implement only [drapbv1alpha4.NodeServer].
-func Start(ctx context.Context, nodeServers []interface{}, opts ...Option) (result DRAPlugin, finalErr error) {
+// options are mandatory. Otherwise only [DriverName] is mandatory.
+func Start(ctx context.Context, plugin DRAPlugin, opts ...Option) (result *Helper, finalErr error) {
 	logger := klog.FromContext(ctx)
 	o := options{
 		logger:        klog.Background(),
 		grpcVerbosity: 6, // Logs requests and responses, which can be large.
-		nodeV1alpha4:  true,
+		serialize:     true,
```

Comment:
Another convention question. Is it common to sort field initialization to match declarations / alphabetically?

Reply:
Definitely not alphabetical. According to declaration makes more sense, although sometimes declarations follow a grouping that might make less for initialization. I would say "whatever looks good" is fine.